microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -609,6 +610,36 @@ To enhance the Kids Mode experience, Microsoft Edge adds a safe keyword to the h
 Kids Mode does not collect data for personalization of the news feed or other Microsoft services.  You may not change the privacy settings for Kids Mode.  Other settings like Windows Defender SmartScreen and diagnostic data are configured according to the profile in which Kids Mode was launched.  For more information about diagnostic data about how you use the browser and Windows Defender SmartScreen, see the [Diagnostic Data](#diagnostic-data) and [SmartScreen](#smartscreen) sections.
 
 
+<!-- ====================================================================== -->
+## Magnify Image
+
+To use the Magnify Image feature, right-click an image and then select **Magnify image**, or press the **Ctrl** key twice while hovering over an image.  The Magnify Image feature retrieves the largest available version of the image from the website and displays it in a popup window.
+
+
+<!-- ------------------------------ -->
+#### AI-based image enhancement
+
+If your browser has the Web Machine Learning Neural Network (WebNN) API enabled, there's also an **AI enhance** button, that enables you to use AI to enhance the image.  To check whether WebNN is enabled, go to the `edge://gpu` page, and then in the **Graphics Feature Status** section, see whether **WebNN** says **Enabled**:
+
+![The status of the WebNN setting, shown in the edge://gpu page](./index-images/webnn-disabled-gpu.png)
+
+To change the WebNN setting, go to `edge://flags`, search for **Enables WebNN API**, and then select **Default**, **Enabled**, or **Disabled**.
+
```

Comment:
Remove because this isn't needed when the feature is enabled by default.

```suggestion
```

Reply:
removed png & lines.  @sjdallst2 - if i should restore anything, let me know